troubadix/troubadix.py

```diff
@@ -163,6 +164,12 @@ def main(args=None):
         first_file = files[0].resolve()
         root = get_root(first_file)
 
+    if parsed_args.plugins_config_file:
+        with open(parsed_args.plugins_config_file, "rb") as file:
+            plugins_config = tomli.load(file)
+    else:
+        plugins_config = {}
```

Comment:
That was done so that the file is seen as optional and Troubadix can be run without it.
I will remove the default values that keep it from crashing. I will set it as a required parameter in argparse.